vars/common.yml

```diff
@@ -25,6 +25,7 @@ supplier_frontend:
   instance_type: t2.micro
   min_instance_count: 1
   max_instance_count: 2
+  password_salt: ResetPasswordSalt
```

Comment:
I'd keep the password_salt in the secret config, just to be on the safe side. Otherwise there's no reason to keep it as environment variable, it can just be set in app config directly